releasetasks/templates/notification/email_release_drivers_task.yml.tmpl

```diff
@@ -1,14 +1,17 @@
 {#
 Emails release-drivers via Pulse-Notify service
 The macro call should look like
-    email_release_drivers_task(product, version, channel, requires, update_channel=None)
+    email_release_drivers_task(product, version, channel, requires, update_channel=None, release_eta=None)
 As defined at the top of release_graph.yml.tmpl
 #}
 
 {% if (product is defined) and (version is defined) and (channel is defined) and (requires is defined) %}
 
     {% set full_channel_name = "{}-{}".format(channel, update_channel) if update_channel else channel %}
     {% set buildername = "release-{}-{}_email_{}".format(branch, product, full_channel_name) %}
+    {% set message = (release_eta is defined and release_eta)
+           and "{} {} {} updates will be published to {} channel on {} <EOM>".format(product, channel, version, full_channel_name, release_eta)
```

Comment:
Nit: I think "scheduled for XXX" is better than "scheduled on XXX". I could be wrong though, in Romanian I'd say the same :)